src/time.rs

```diff
@@ -199,3 +199,55 @@ pub fn clock_getcpuclockid(pid: Pid) -> Result<ClockId> {
         Err(Errno::from_i32(ret))
     }
 }
+
+#[cfg(any(
+    linux_android,
+    solarish,
+    target_os = "freebsd",
```

Comment:
note: once [this is merged](https://github.com/rust-lang/libc/pull/3509), you could change to `freebsdlike`

Reply:
done